scripts/qemu-check.sh

```diff
@@ -95,6 +95,12 @@ do
 		SHM_MBOX=qemu-bridge-hp-sram-mem
 		ROM="-r ../sof.git/build_${j}_gcc/src/arch/xtensa/rom-$j.bin"
 	fi
+        if [ $j == "imx8" ]
+        then
+                READY_IPC="00 00 00 00 00 00 04 c0"
+                SHM_IPC_REG=qemu-bridge-mu-io
+                SHM_MBOX=qemu-bridge-mbox-io
+        fi
```

Comment:
You have spaces here instead of tabs ;) Everything else is ok.

Reply:
@jajanusz done.